src/inverted-index.js

```diff
@@ -72,31 +80,79 @@ class InvertedIndex {
       });
       // Sort the index keys
```

Comment:
### Critical

@andela-rbabalola

Lines `64 - 69` are actually doing validation, not creating the index, they should be abstracted into it's own method.

Line `71` only takes into cognisance the `text` of the document, if a user types in a search query that includes only the words in the document's `title`, they won't get results, was that your intention?